tests/teamcity/run-server.sh

```diff
@@ -20,7 +20,7 @@ source $DIRNAME/$FXA_TEST_NAME
 
 # optionally, GIT_COMMIT can be set in the environment to override
 if [ -z "$GIT_COMMIT" ]; then
-  GIT_COMMIT=$(curl -s "$FXA_CONTENT_ROOT/ver.json" | jsawk  "return this.commit")
+  GIT_COMMIT=$(curl -s "$FXA_CONTENT_VERSION" | jsawk  "return this.commit" | perl -pe 's/OUT:\s+//')
```

Comment:
(Side note: Ah, I see. If you install `jsawk` with `npm install` you get the core JS file running and emitting the extra `OUT: ` or `ERR: `. But if you get it from some package system you (maybe) get a bash script that wraps the JS, and fixes up those prefixes as a very sophisticated form of error handling (as opposed to just using those crazy Unix exit codes).